src/components/Search/Search.js

```diff
@@ -2,24 +2,41 @@ import React, { useEffect, useState } from 'react';
 import SearchIcon from '@material-ui/icons/Search';
 import InputBase from '@material-ui/core/InputBase';
 import { FormattedMessage } from 'react-intl';
-import { Link, useHistory } from 'react-router-dom';
-import useStyles from './searchStyles';
+import { useHistory } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+
+import ListItemAvatar from '@material-ui/core/ListItemAvatar';
+import Avatar from '@material-ui/core/Avatar';
+
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
 import fuse from './FuseSearch';
-import Loader from '../Loader/Loader';
+import useStyles from './searchStyles';
+
+const useStylesMore = makeStyles(theme => ({
+  root: {
+    position: 'relative',
+  },
+  dropdown: {
+    position: 'absolute',
+    top: 28,
+    right: 0,
+    left: 0,
+    zIndex: 1,
+    border: '1px solid',
+    padding: theme.spacing(1),
+    backgroundColor: theme.palette.background.paper,
+  },
+}));
 
 function Search() {
+  const moreClasses = useStylesMore();
```

Comment:
why more classes? we already have a separate style for this component?

Reply:
Those styles were unnecessary. Removed 'em :)